docs/vtex-io/App-Guides/ab-tests/vtex-io-documentation-running-native-ab-testing.md

```diff
@@ -136,6 +136,8 @@ Before completing your A/B test, it is important to understand the comparative a
 | **Expected Loss**            | Anticipated percentage of conversion loss for the store if the lower conversion rate workspace is selected as the winner (based on **Conversion** results). |
 | **N. of Sessions**           | Total number of sessions for each workspace since the beginning of the test.                        |
 | **N. of Sessions (last 24hrs)** | Number of sessions for each workspace in the past 24 hours during the test.                           |
+| **Revenue**             | Total value of orders placed by users navigating in the version of your store associated to each workspace since the beginning of the test (according to the currency configured in the store).|
```

Comment:
```suggestion
| **Revenue**             | Total value of orders placed by users navigating the store version associated with each workspace since the beginning of the test (expressed in the store's native currency).|
```

Reply:
- Sugestão para deixar a descrição mais concisa.
- "associated to" é aceitável, mas o mais correto é ["associated with"](https://www.oxfordlearnersdictionaries.com/us/definition/english/associated) segundo os dicionários.
- Sugestão para utilizar "expressed in" ao invés de "according to" para deixar mais explícito que estamos falando sobre como essa unidade é expressa